tests/Feature/Http/Controllers/TransactionsControllerTest.php

```diff
@@ -76,7 +74,7 @@
 it('should show the correct decimal places for the stats', function ($decimalPlaces, $amount, $fee, $expectedFormattedFee) {
     $this->travelTo('2021-04-14 16:02:04');
 
-    $gasUsed = 21000;
+    $gasUsed = 210000;
```

Comment:
```suggestion
    $gasUsed = 21000;
```

for transfers the usual amount is 21000, not sure why this needed changing 🤔

Reply:
@ItsANameToo a typo while I was trying to fix the test, should be fine now